ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -156,6 +156,19 @@ open class NavigationBar: UINavigationBar {
         return nil
     }
 
+    @objc public func barButtonItemView(tag: Int) -> UIView? {
```

Comment:
barbuttontiemview(with tag: Int) might be more swift friendly naming

Reply:
please add documentation

Reply:
Done